src/test/Component.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
- [ ] Using the React testing library, kindly add tests that simulate user interactions like clicking buttons and expecting the app to behave a certain way based on the action e.g. performing a calculation, or clicking a navbar link.